neutone_sdk/core.py

```diff
@@ -264,7 +321,7 @@ def get_output_gain_default_value(self) -> float:
     def get_core_preserved_attributes(self) -> List[str]:
         return [
             "model",  # nn.Module
-            "default_param_values",  # Registered buffer
```

Comment:
So you've tested that removing this doesn't break the plugin in some way?

Reply:
yeah this is redundant since the corresponding getter method is being exported already